lib/cadet/assessments/answer.ex

```diff
@@ -63,6 +63,11 @@ defmodule Cadet.Assessments.Answer do
     |> validate_xp_grade_adjustment_total()
   end
 
+  @spec comment_changeset(%__MODULE__{} | Ecto.Changeset.t(), map()) :: Ecto.Changeset.t()
+  def comment_changeset(answer, params) do
```

Comment:
Can we write a TODO to refactor this in the future. This is a very unsafe changeset function. So far, as long as you use it to change the `comment` it should be fine.

Reply:
Added a todo